fix: This makes the indeterminate HTML attribute available for use. Fixes #9176. (#9180)
sveltejs/svelte: .changeset/ten-poems-remember.md

---
'svelte': patch
---

> This allows a User to set the indeterminate HTML Attribute by adding it to the list of HTMLAttributes.

Reviewer: ```suggestion
fix: add `indeterminate` to the list of HTMLAttributes
```
